anvil/anvil_test.go

```diff
@@ -15,7 +15,21 @@ import (
 )
 
 func TestAnvil(t *testing.T) {
-	cfg := config.ChainConfig{ChainID: 10, Port: 0}
+	t.Run("unset default host", func(t *testing.T) {
+		cfg := config.ChainConfig{ChainID: 10, Port: 0}
+		testAnvilInstance(t, cfg)
+	})
```

Comment:
this run should fail because anvil correctly expects a host.

can delete this test